.github/ISSUE_TEMPLATE/major_change.md

```diff
@@ -7,39 +7,35 @@ assignees: ''
 
 ---
 
-# What is this issue?
+# Proposal
 
-This is a **major change proposal**, which means a proposal to make a notable change to the compiler -- one that either alters the architecture of some component, affects a lot of people, or makes a small but noticeable public change (e.g., adding a compiler flag). You can read more about the MCP process on https://forge.rust-lang.org/.
+*Insert your proposal here, overwriting this text.*
```

Comment:
Would it be worth explicitly saying something like: "The proposal should not just be a link to a document hosted elsewhere; you can develop a proposal collaboratively using such a mechanism, and you can include a link to a current version if so, but the content of the proposal should be pasted inline."